plugins/permissions/src/interactions/mod.ts

```diff
@@ -6,6 +6,14 @@ export function sendInteractionResponse(bot: BotWithCache) {
   const sendInteractionResponseOld = bot.helpers.sendInteractionResponse;
 
   bot.helpers.sendInteractionResponse = function (id, token, options) {
+    if (options.data?.title !== undefined) {
```

Comment:
```suggestion
    if (options.data?.title) {
```

Reply:
What if title is `""`

Reply:
oh i see. TBH i woulda preferred .title?.length >= 0 but whatever this works also